components/sidebar/channel_navigator/index.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch, ActionCreatorsMapObject} from 'redux';
+
+import {GlobalState} from 'mattermost-redux/types/store';
+import {ActionFunc} from 'mattermost-redux/types/actions';
+
+import {openModal} from 'actions/views/modals';
+import {browserHistory} from 'utils/browser_history';
+
+import ChannelNavigator from './channel_navigator';
+
+// TODO: For Phase 1. Will revisit history in Phase 2
+function goBack() {
+    return (dispatch: any, getState: any) => {
```

Comment:
My assumption was they would be changed to use Redux, but since we decided to push that history bit to Phase 2, it didn't happen.

I'd like to leave these as is until we finalize the solution, but I will change them if need be.

Reply:
Sure okay, Sounds fine.

Reply:
I'm a fan of doing it this way since it means that the code calling this doesn't need to be aware that the functionality for this will change in the future.